components/horizontal-video-testimonials/manifest.json

```diff
@@ -0,0 +1,194 @@
+              "bgImage": {
+                "title": "Background image",
+                "description": "Select a background image for the section. A dark overlay will be applied over the image.",
+                "type": "string",
+                "format": "matrix-asset-uri"
+              },
+              "marginTop": {
+                "title": "Spacing above (top margin)",
+                "description": "Add spacing above the section. 'Base' is the smallest with '10' (default) being the largest option. Selecting 'default' will use the default site-wide vertical spacing between components.",
+                "type": "string",
+                "default": "10",
+                "enum": [
+                  "default",
+                  "base",
+                  "1",
+                  "2",
+                  "3",
+                  "4",
+                  "5",
+                  "6",
+                  "7",
+                  "8",
+                  "9",
+                  "10"
+                ]
+              },
+              "marginBottom": {
+                "title": "Spacing below (bottom margin)",
+                "description": "Add spacing below the section. 'Base' is the smallest with '10' (default) being the largest option. Selecting 'default' will use the default site-wide vertical spacing between components.",
+                "type": "string",
+                "default": "10",
+                "enum": [
+                  "default",
+                  "base",
+                  "1",
+                  "2",
+                  "3",
+                  "4",
+                  "5",
+                  "6",
+                  "7",
+                  "8",
+                  "9",
+                  "10"
+                ]
+              }
+            }
+          },
+          "testimonials": {
+            "title": "Video Testimonials",
+            "description": "Enter the details for each video testimonial card.",
+            "type": "array",
+            "minItems": 1,
+            "items": {
+              "type": "object",
+              "properties": {
+                "heading": {
+                  "title": "Card heading",
+                  "type": "string"
+                },
+                "description": {
+                  "title": "Description",
+                  "description": "This is the description/body copy.",
+                  "type": "FormattedText"
+                },
+                "videoImage": {
+                  "title": "Video preview image",
+                  "description": "Select an image to use as the video preview. The aspect ratio of the image displayed will be 16x9.",
+                  "type": "string",
+                  "format": "matrix-asset-uri"
+                },
+                "youtubeId": {
+                  "title": "Youtube video ID",
+                  "description": "Enter the video ID from YouTube.",
+                  "type": "string"
+                },
```

Comment:
Supporting Youtube ID - I'm reusing the CardThumbnail package which seems to only support youtube videos in the modal